website/components/LandingPage/index.tsx

```diff
@@ -214,9 +215,16 @@ export default function LandingPage(): ReactElement {
   const onClickLoad = (appProps: AppDataProps): void => {
     // TODO: Make URL search params from the appProps and append it to the viewer URL so the URL can be shared directly.
     // Alternatively, AppWrapper should manage syncing URL and viewer props.
-    navigation("viewer", {
-      state: appProps,
-    });
+    const url = appProps.imageUrl;
+    if (url instanceof Array) {
```

Comment:
I believe this style of check is generally preferred, but you may know something I don't:


```suggestion
    if (Array.isArray(url)) {
```